src/host/polyfills/simpleView.ts

```diff
@@ -94,3 +105,32 @@ export function applyInspectorCommonCssPatch(content: string, isRelease?: boolea
         `${css}${separator} $1`,
     );
 }
+
+export function applyInspectorDarkCssPatch(content: string, isRelease?: boolean) {

```

Comment:
This can replace applyInspectorCommonCssPatch also, but I prefer to leave it separate as multipurpose this behavior will probably lead to some errors in the future.  Thoughts?